studio/gcloud_worker.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, runner_args=None, zone='us-central1-f',
         self.logger = logging.getLogger("GCloudWorkerManager")
         self.logger.setLevel(verbose)
         self.auth_cookie = auth_cookie
+        self.user_startup_script = user_startup_script
```

Comment:
Is this the user specified startup script.  If so can we get rid of them they are harmful for the data center runner.  Thanks

Reply:
Yes we can; will do

Reply:
Lets not remove it so fast, lets make it deprecated first.